### DISCUSSION TITLE
french traslation became in englsih

### DISCUSSION THREAD

**Discussion by raphaelmarie** [DivinumOfficium/divinum-officium#3781] [General]
Hello Bonjour

A la fin des Laudes "Divino Alfflatu", dans la partie française, la traduction en français est devenue en anglais : 
" V. Que le Seigneur nous donne Sa paix.
R. Et la vie éternelle. Ainsi soit-il.

V. May the Lord grant us his peace
R. And life everlasting.

Pourquoi ?

At the end of the Laudes "Divino Afflatu", in the french part. The french translation became en english language :
"V. Que le Seigneur nous donne Sa paix.
R. Et la vie éternelle. Ainsi soit-il.
V. May the Lord grant us his peace
R. And life everlasting.

Why ?

**Comment by FAJ-Munich** [comment]
this pull request https://github.com/DivinumOfficium/divinum-officium/pull/3782 should close this error

**Comment by APMarcello3** [comment]
https://github.com/DivinumOfficium/divinum-officium/commit/2b277ce73cf64e1c39840e160ebd4dbbcba153e4
